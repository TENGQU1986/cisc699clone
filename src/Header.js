import React from 'react'
import './header.css'

const Header = ({ user }) => {
    return (
        <div className="header">
            <span>{user?.displayName}</span>
            <span>Websites Performance Analysis Forum</span>
            <span>{user?.email}</span>
        </div>
    )
}

export default Header
