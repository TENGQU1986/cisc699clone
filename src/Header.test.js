import React from 'react';
import Header from './Header';
import { render, fireEvent, act } from "@testing-library/react";


it('should render Header component', () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
})