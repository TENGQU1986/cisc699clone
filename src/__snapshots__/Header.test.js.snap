// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header component 1`] = `
<div>
  <div
    class="header"
  >
    <span />
    <span />
  </div>
</div>
`;
